packages/common/src/api/tan-query/useUserAlbums.ts

```diff
@@ -0,0 +1,55 @@
+import { useQuery, useQueryClient } from '@tanstack/react-query'
+import { useDispatch } from 'react-redux'
+
+import { userCollectionMetadataFromSDK } from '~/adapters/collection'
+import { transformAndCleanList } from '~/adapters/utils'
+import { useAppContext } from '~/context/appContext'
+import { Id, OptionalId } from '~/models/Identifiers'
+
+import { QUERY_KEYS } from './queryKeys'
+import { useCurrentUserId } from './useCurrentUserId'
+import { primeCollectionData } from './utils/primeCollectionData'
+
+type GetAlbumsOptions = {
+  userId: number | null
+  limit?: number
+  offset?: number
+}
+
+type Config = {
+  staleTime?: number
+  enabled?: boolean
+}
+
+export const useUserAlbums = (options: GetAlbumsOptions, config?: Config) => {
+  const { audiusSdk } = useAppContext()
```

Comment:
we moved to `audiusQueryContext`, right?

Reply:
ah right